docs/CONFIG.md

```diff
@@ -2776,16 +2776,16 @@ Enabled = true
 BlockDelay = 1
 
 [GasEstimator]
-Mode = 'FixedPrice'
-PriceDefault = '1 micro'
-PriceMax = '1 micro'
-PriceMin = '1 micro'
-LimitDefault = 7000000
+Mode = 'BlockHistory'
+PriceDefault = '20 gwei'
+PriceMax = '100 micro'
+PriceMin = '1 gwei'
```

Comment:
These defaults should be functional, but we should check if they can be improved before merging.